.changeset/lemon-donuts-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-plugin-api': minor
```

Comment:
```suggestion
'@backstage/backend-plugin-api': patch
```

Not a breaking change since there behavior hasn't changed. It wasn't possible to pass plugin scoped services before either